Sources/MapboxNavigation/TopBannerViewController.swift

```diff
@@ -71,19 +71,10 @@ open class TopBannerViewController: UIViewController {
         return banner
     }()
     
-    lazy var lanesView: LanesView = .forAutoLayout(hidden: true)
-    lazy var nextBannerView: NextBannerView = .forAutoLayout(hidden: true)
-    lazy var statusView: StatusView = {
-        let view: StatusView = .forAutoLayout()
-        view.isHidden = true
-        return view
-    }()
-    
-    lazy var junctionView: JunctionView = {
-        let view: JunctionView = .forAutoLayout()
-        view.isHidden = true
-        return view
-    }()
+    public var lanesView: LanesView = .forAutoLayout(hidden: true)
+    public var nextBannerView: NextBannerView = .forAutoLayout(hidden: true)
+    public var statusView: StatusView = .forAutoLayout(hidden: true)
+    public var junctionView: JunctionView = .forAutoLayout(hidden: true)
```

Comment:
These public properties should be documented.

Reply:
Thanks for the suggestion, I'll add the documentation for the public properties